machine-learning/_posts/2018-01-15-translearning.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: review
+title:  "Unsupervised Transformation Learning
+via Convex Relaxations"
+tags:   unsupervised
+author: Frédéric Branchaud-Charron
+pdf: http://papers.nips.cc/paper/7263-unsupervised-transformation-learning-via-convex-relaxations.pdf
+cite:
+  authors: "Tatsunori B., Hashimoto John C. and Duchi Percy Liang"
+  title:   "Unsupervised Transformation Learning
+via Convex Relaxations"
+  venue:   "NIPS 2017"
+---
+
+The authors propose an unsupervised method to estimate the transformation matrix to describe a dataset.
```

Comment:
Suggestion: "method to discover transformation matrices associated to relationships in the dataset."